tests/index.test.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import renderer from 'react-test-renderer'
+import {render, screen} from '@testing-library/react'
+import Index from '../pages/index'
+
+describe("Exemplar-react-test renderer Snapshot test", () => {
+  it('renders homepage unchanged', () => {
+    const tree = renderer.create(<Index />).toJSON()
+    expect(tree).toMatchSnapshot()
+  })
+})
+
+
+describe("Example react testing-library Test", () => {
```

Comment:
if this is testing a subset of the snapshot matching test above, and it's here to serve as an example rather than test something additional, maybe make it `xit` and comment it as an example?

Reply:
oh neat! TIL `xit`